package.json

```diff
@@ -45,7 +45,9 @@
   "module": "dist/es/src/index.js",
   "dependencies": {
     "babel-runtime": "~6.25.0",
-    "color": "^3.1.3"
+    "color": "^3.1.3",
+    "lodash": "^4.17.20",
```

Comment:
Do we really need `lodash`, just for a single `omit` method ?
```ts
const { x, y, ...reset } = object;
```
should do the same trick 🤔

Reply:
Oh right